help/androidpublisher.md

```diff
@@ -42,67 +44,67 @@ https://developers.google.com/android-publisher/getting_started
                 KeystorePassword = "my password"
                 KeystoreAlias = "my key alias"
             })
-        |> fun file -> file.CopyTo(Path.Combine(androidProdDir, file.Name)) |> ignore
+        |> Seq.iter(fun file -> file.CopyTo(Path.Combine(androidProdDir, file.Name)) |> ignore)
```

Comment:
Yes, AndroidSignAndAlign returns multiple FileInfo instead of only one, because we can have multiple APKs.

Do you prefere anothter fonction to preserve existing build scripts ?